README.md

```diff
@@ -126,13 +147,20 @@ kafkaSource.properties.sasl.mechanism=
 kafkaSource.properties.sasl.jaas.config=
 ```
 ```
-#Spark yarn sink properties. Properties used to deploy and run Spark job in Yarn. Not all are required. Adjust according to your use case.
-spark.submit.thread.pool.size=
-sparkYarnSink.hadoopResourceManagerUrlBase=
+# Recurring sensor properties.
+recurring-sensor.maxJobsPerDuration=8
+recurring-sensor.duration=1h
```

Comment:
I would change this to `recurringSensor` for consistency with `kafkaSource`. In the prefix annotation in `RecurringSensorConfig`, I have to use kebab case (as described here: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.external-config.typesafe-configuration-properties.relaxed-binding), but camel case can be used in the application property

Reply:
Done